pkgs/servers/sql/postgresql/ext/tsja.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, fetchurl
+, nixosTests
+, stdenv
+
+, mecab
+, postgresql
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tsja";
+  version = "0.5.0";
+
+  src = fetchurl {
+    url = "https://www.amris.jp/tsja/tsja-${version}.tar.xz";
+    hash = "sha256-Z8iwMQwS5mjnx8D8hgHB/5rxrTehrXhBRFcSUJzUAGI=";
+  };
+
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace /usr/local/pgsql ${postgresql} \
+      --replace -L/usr/local/lib "" \
+      --replace -I/usr/local/include ""
+    substituteInPlace tsja.c --replace /usr/local/lib/mecab ${mecab}/lib/mecab
+  '';
+
+  buildInputs = [ mecab postgresql ];
+
+  installPhase = ''
+    mkdir -p $out/lib $out/share/postgresql/extension
+    mv libtsja.so $out/lib
+    mv dbinit_libtsja.txt $out/share/postgresql/extension/libtsja_dbinit.sql
```

Comment:
Could you make this file executable? I'm not sure why, but all other postgres packages i look at mark these executable.

Reply:
Maybe not explicitly in the builders, but every package build I've inspected has executable `.sql` files. Try for example:

    fd '\.sql$' $(nix-build . -A postgresql14Packages.smlar --no-out-link) --type x

I've been search for why this might be the case, but I have not gotten any wiser. I don't consider this a blocker.